tools/Test/SmokeTest/RmCoreSmokeTests.ps1

```diff
@@ -5,6 +5,11 @@
 # $debugState = $false
 [cmdletbinding()]
 param(
+    [string]
+    [Parameter(Mandatory = $false, Position = 0)]
```

Comment:
I think subscription ID should not be a parameter of smoke test, because

1. Smoke test is based on an asumption that you have already set up the environment.
1. `-subscriptionId` does not always work. It only works when you didn't login, which may cause confusion.

Is there any specific reason why you add it? If not, I think we can simply undo changes to this file

Reply:
Subscription Id is used only when you didn't sign in before running smoke tests. That is not true. So we can safely remove this parameter